include/nbl/video/SPhysicalDeviceLimits.h

```diff
@@ -0,0 +1,497 @@
+    //VkBool32              standardSampleLocations;
+    //VkDeviceSize          optimalBufferCopyOffsetAlignment;
+    //VkDeviceSize          optimalBufferCopyRowPitchAlignment;
+    uint64_t nonCoherentAtomSize;
+            
+    /* Vulkan 1.1 Core  */
+    uint32_t subgroupSize;
+    core::bitflag<asset::IShader::E_SHADER_STAGE> subgroupOpsShaderStages;
+    bool shaderSubgroupBasic = false;
+    bool shaderSubgroupVote = false;
+    bool shaderSubgroupArithmetic = false;
+    bool shaderSubgroupBallot = false;
+    bool shaderSubgroupShuffle = false;
+    bool shaderSubgroupShuffleRelative = false;
+    bool shaderSubgroupClustered = false;
+    bool shaderSubgroupQuad = false;
+    bool shaderSubgroupQuadAllStages = false; //quadOperationsInAllStages;
+    //VkPointClippingBehavior    pointClippingBehavior;
+    //uint32_t                   maxMultiviewViewCount;
+    //uint32_t                   maxMultiviewInstanceIndex;
+    //VkBool32                   protectedNoFault;
+    //uint32_t                   maxPerSetDescriptors;
+    //VkDeviceSize               maxMemoryAllocationSize;
+            
+    /* Vulkan 1.2 Core  */
+    //VkShaderFloatControlsIndependence    denormBehaviorIndependence;
+    //VkShaderFloatControlsIndependence    roundingModeIndependence;
+    //VkBool32                             shaderSignedZeroInfNanPreserveFloat16;
+    //VkBool32                             shaderSignedZeroInfNanPreserveFloat32;
+    //VkBool32                             shaderSignedZeroInfNanPreserveFloat64;
+    //VkBool32                             shaderDenormPreserveFloat16;
+    //VkBool32                             shaderDenormPreserveFloat32;
+    //VkBool32                             shaderDenormPreserveFloat64;
+    //VkBool32                             shaderDenormFlushToZeroFloat16;
+    //VkBool32                             shaderDenormFlushToZeroFloat32;
+    //VkBool32                             shaderDenormFlushToZeroFloat64;
+    //VkBool32                             shaderRoundingModeRTEFloat16;
+    //VkBool32                             shaderRoundingModeRTEFloat32;
+    //VkBool32                             shaderRoundingModeRTEFloat64;
+    //VkBool32                             shaderRoundingModeRTZFloat16;
+    //VkBool32                             shaderRoundingModeRTZFloat32;
+    //VkBool32                             shaderRoundingModeRTZFloat64;
+    //uint32_t                             maxUpdateAfterBindDescriptorsInAllPools;
+    //VkBool32                             shaderUniformBufferArrayNonUniformIndexingNative;
+    //VkBool32                             shaderSampledImageArrayNonUniformIndexingNative;
+    //VkBool32                             shaderStorageBufferArrayNonUniformIndexingNative;
+    //VkBool32                             shaderStorageImageArrayNonUniformIndexingNative;
+    //VkBool32                             shaderInputAttachmentArrayNonUniformIndexingNative;
```

Comment:
true if `NV_gpu_shader5` is present, false otherwise

Reply:
actually also `GL_EXT_nonuniform_qualifier`

Reply:
Yes but I think `shaderInputAttachmentArrayNonUniformIndexingNative` should be false?

Reply:
`NAtive` would be true with `NV_gpu_shader5`

Reply:
I'm emphasizing on the `InputAttachment` part

Reply:
ah yeah, the input attachment should be false ;)